### PR TITLE
added CLI args, set up template for 3 models and start of the testing…

### PR DESCRIPTION
**Pull request by anarasimham**
… process

### CODE REVIEW

**Comment by sfwatanabe** [comment]
General thought - we might want to go with the yaml config approach since we'll have different hyperparameters across model types.

**Comment by anarasimham** [comment]
Fair point, I can change to yaml

Ashish

File: config.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+mode:
```
Comment (killyosaur) [inline config.yml:1]: Is the idea here to set the order? not sure why this is a list.

File: config.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  - train
+  - eval
+
+model_types:
```
Comment (killyosaur) [inline config.yml:5]: This should probably be a value not a list. I assume We will Create different configs for each grithm

Comment (killyosaur) [reply config.yml:5]: otherwise, if the goal is to turn one off, Just set up a `train: true` and `eval: true`

Comment (killyosaur) [reply config.yml:5]: looking at the run file, I think this section is unnecessary. we can just run or not run a model using an enabled: true in each model definition section. if we want to loop things, we can use the hard coded array to find each section.

**Review by killyosaur — APPROVED** [review]

**Pull request merged**